test/fixtures/rules/object-spread/object-spread.js

```diff
@@ -0,0 +1,10 @@
+// `prefer-object-spread/prefer-object-spread` - Require use of object spread instead of object.assign
```

Comment:
Following my suggestion to remove `rules/object-spread`, please create a `contrived.bad.js` inside https://github.com/moxystudio/eslint-config/tree/master/test/fixtures/samples/es7-object-spread